### PR TITLE
[Choreo] Addition of info log to recognise the rate-limit level

### PR DESCRIPTION
**Pull request by slahirucd7**
### Purpose
$subject

### Issues
https://github.com/wso2-enterprise/choreo/issues/26517

### Automation tests
 - Unit tests added: Yes/No
 - Integration tests added: Yes/No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: adapter/internal/discovery/xds/rate_limiter_cache.go [hunk 1]
```diff
@@ -88,6 +88,11 @@ func (r *rateLimitPolicyCache) AddAPILevelRateLimitPolicies(apiID string, mgwSwa
 		return nil
 	}
 	level := strings.ToUpper(mgwSwagger.RateLimitLevel)
+	if mgwSwagger.RateLimitLevel == envoyconf.RateLimitPolicyAPILevel {
+		loggers.LoggerXds.Infof("API level rate-limiting enabled for the API: %q", apiID)
+	} else {
+		loggers.LoggerXds.Infof("Operation level rate-limiting enabled for the API: %q", apiID)
+	}
```
Comment (renuka-fernando) [inline adapter/internal/discovery/xds/rate_limiter_cache.go:95]:
```suggestion
	loggers.LoggerXds.Infof("Rate-limiting enabled for the API: %q, level: %q", apiID, level)
```

**Review by renuka-fernando — APPROVED** [review]

**Review by malinthaprasan — APPROVED** [review]

File: adapter/internal/discovery/xds/rate_limiter_cache.go [hunk 2]
```diff
@@ -88,6 +88,7 @@ func (r *rateLimitPolicyCache) AddAPILevelRateLimitPolicies(apiID string, mgwSwa
 		return nil
 	}
 	level := strings.ToUpper(mgwSwagger.RateLimitLevel)
+	loggers.LoggerXds.Infof("Rate-limiting enabled for the API: %q, level: %q", apiID, level)
```
Comment (malinthaprasan) [inline adapter/internal/discovery/xds/rate_limiter_cache.go:91]: This is not logged during API calls right? Logged only during a deployment?

Comment (renuka-fernando) [reply adapter/internal/discovery/xds/rate_limiter_cache.go:91]: Yes, during deployment

Comment (slahirucd7) [reply adapter/internal/discovery/xds/rate_limiter_cache.go:91]: This method is being called during the API deployment time. Hence log will not be printed for the API invocations.

**Comment by choreo-cicd** [comment]
[failed] Controlplane cluster : dev-deployment-v2 : 20240130.13

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(NorthEU) cluster : dev-deployment-v2 : 20240130.13

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(EastUS) cluster : dev-deployment-v2 : 20240130.13

**Comment by choreo-cicd** [comment]
[] Controlplane cluster : dev-deployment-v2 : 20240130.31

**Comment by choreo-cicd** [comment]
[] Dataplane(EastUS) cluster : dev-deployment-v2 : 20240130.31

**Comment by choreo-cicd** [comment]
[] Dataplane(NorthEU) cluster : dev-deployment-v2 : 20240130.31

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(NorthEU) cluster : stage-deployment-v2 : 20240201.1

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(EastUS) cluster : stage-deployment-v2 : 20240201.1

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : stage-deployment-v2 : 20240201.1

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : prod-deployment-v2 : 20240201.3

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(EastUS) cluster : prod-deployment-v2 : 20240201.3

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(NorthEU) cluster : prod-deployment-v2 : 20240201.3

**Pull request merged**